### PR TITLE
[OSCD] Verclsid.exe Runs COM Object (LOLBin) 

### PR DESCRIPTION
**Pull request by stvetro**
Task 88 from #1014

### CODE REVIEW

**Review by w0rk3r — COMMENTED** [review]
Hey @stvetro, just reviewed and the rule looks ready except for not having any tag, consider adding tags like:

`- attack.defense_evasion`
`- attack.t1218`
Thanks for your contribution

File: rules/windows/process_creation/win_verclsid_runs_com.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+date: 2020/10/09
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    image_path:
+        Image|endswith: 'verclsid.exe'
+    cmd_s:
+        CommandLine|contains: '/S'
+
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_verclsid_runs_com.yml:19]: Hello @stvetro! Thank you for your contribution! Could you please remove the empty line here? It's gonna beat yaml structure

File: rules/windows/process_creation/win_verclsid_runs_com.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    - https://gist.github.com/NickTyrer/0598b60112eaafe6d07789f7964290d5
+    - https://bohops.com/2018/08/18/abusing-the-com-registry-structure-part-2-loading-techniques-for-evasion-and-persistence/
+author: Victor Sergeev, oscd.community
+date: 2020/10/09
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    image_path:
+        Image|endswith: 'verclsid.exe'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_verclsid_runs_com.yml:16]: Could you please also add `\` to the beginning of the `verclsid.exe`? It's gonna improve the performance of the search

Comment (stvetro) [reply rules/windows/process_creation/win_verclsid_runs_com.yml]: Thanks for advice! both done

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_verclsid_runs_com.yml]: thank you!

**Comment by yugoslavskiy** [comment]
Thank you very much, @stvetro!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**